examples/llm/hotpot_qa.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import os
+from itertools import chain
+
+import datasets
+import torch
+from tqdm import tqdm
+
+from torch_geometric import seed_everything
+from torch_geometric.loader import RAGQueryLoader
+from torch_geometric.nn.nlp import TXT2KG, SentenceTransformer
+from torch_geometric.utils.rag.backend_utils import (
+    create_remote_backend_from_triplets,
+    make_pcst_filter,
+    preprocess_triplet,
+)
+from torch_geometric.utils.rag.feature_store import (
+    SentenceTransformerFeatureStore,
+)
+from torch_geometric.utils.rag.graph_store import NeighborSamplingRAGGraphStore
+
+if __name__ == '__main__':
+    seed_everything(50)
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--NV_NIM_MODEL', type=str,
+                        default="nvidia/llama-3.1-nemotron-70b-instruct")
+    parser.add_argument('--NV_NIM_KEY', type=str, default="")
+    parser.add_argument('--local_lm', action="store_true")
+    parser.add_argument('--percent_data', type=float, default=1.0)
+    parser.add_argument('--chunk_size', type=int, default=512)
```

Comment:
Would be nice to have example-level descriptions for the less obvious args, like `chunk_size`